content/en/news/releases/1.17.x/announcing-1.17.1/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Announcing Istio 1.17.1
+linktitle: 1.17.1
+subtitle: Patch Release
+description: Istio 1.17.1 patch release.
+publishdate: 2023-02-21
+release: 1.17.1
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.17.0 and Istio 1.17.1.
+
```

Comment:
Looking at the 1.15 and 1.16 release notes, we also want to call out the Go update:

This release includes security fixes in Go 1.20.1 (released 2023-02-14) for the `crypto/tls`, `mime/multipart`, `net/http`, and `path/filepath` packages.

Reply:
From: https://go.dev/doc/devel/release